win32/src/sync_win32.c

```diff
@@ -6,16 +6,17 @@
 
 IMPLEMENT_MOCKABLE_FUNCTION(, bool, wait_on_address, volatile_atomic int32_t*, address, int32_t*, compare_address, uint32_t, timeout_ms)

```

Comment:
I included them in sync.h. Should't that be sufficient?

---
In reply to: [443824534](https://github.com/Azure/azure-c-pal/pull/6#discussion_r443824534) [](ancestors = 443824534)

Reply:
We always try to include what we use in the file, rather than relying upon another include from somewhere else.

---
In reply to: [443887001](https://github.com/Azure/azure-c-pal/pull/6#discussion_r443887001) [](ancestors = 443887001,443824534)